core/editor/editor.php

```diff
@@ -719,10 +728,11 @@ public function enqueue_styles() {
 			);
 		}
 
-		if ( Responsive::has_custom_breakpoints() ) {
-			$breakpoints = Responsive::get_breakpoints();
-
-			wp_add_inline_style( 'elementor-editor', '.elementor-device-tablet #elementor-preview-responsive-wrapper { width: ' . $breakpoints['md'] . 'px; }' );
+		/** @var Breakpoint $mobile_portrait_breakpoint */
+		$mobile_portrait_breakpoint = Plugin::$instance->breakpoints->get_breakpoints( Breakpoints_Manager::BREAKPOINT_KEY_MOBILE );
+		// What we want to know is whether the md (mobile portrait) breakpoint has the same value as the default or not.
+		if ( $mobile_portrait_breakpoint->is_custom() ) {
+			wp_add_inline_style( 'elementor-editor', '.elementor-device-tablet #elementor-preview-responsive-wrapper { width: ' . $mobile_portrait_breakpoint->get_value() . 'px; }' );
```

Comment:
```suggestion
			wp_add_inline_style( 'elementor-editor', '.elementor-device-tablet #elementor-preview-responsive-wrapper { width: ' . Plugin::$instance->breakpoints->get_device_min_value( Breakpoints_Manager::BREAKPOINT_KEY_TABLET ) . 'px; }' );
```

Reply:
Done. Also, it is necessary to not only check if the mobile breakpoint has a custom value, but also if the `mobile_extra` breakpoint is enabled. If it is, the min breakpoint for tablet will be based on `mobile_extra` and not `mobile`.